### PR TITLE
Add Appointment

### PR DESCRIPTION
**Pull request by clarlzx**

### CODE REVIEW

**Review by nicoleang09 — APPROVED** [review]
Changes look good to be merged!

File: src/main/java/seedu/address/storage/Storage.java [hunk 1]
```diff
@@ -29,4 +30,57 @@
...
+//    @Override
+//    Optional<ReadOnlyTBook> readTBook() throws DataConversionException, IOException;
+//
+//    @Override
+//    void saveTBook(ReadOnlyTBook tBook) throws IOException;
+//
+//    @Override
+//    void saveTBook(ReadOnlyTBook tBook, Path filePath) throws IOException;
+//
+//}
```
Comment (Stratostorm) [inline src/main/java/seedu/address/storage/Storage.java:86]: Whats all this stuff? Can just delete?

File: src/test/java/seedu/address/storage/JsonAppointmentBookStorageTest.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    //        jsonAppointmentBookStorage.saveAppointmentBook(original, filePath);
+    //        readBack = jsonAppointmentBookStorage.readAppointmentBook(filePath).get();
+    //        assertEquals(original, new AppointmentBook(readBack));
+    //
+    //        // Save and read without specifying file path
+    //        original.addPerson(IDA);
+    //        jsonAppointmentBookStorage.saveAppointmentBook(original); // file path not specified
+    //        readBack = jsonAppointmentBookStorage.readAppointmentBook().get(); // file path not specified
+    //        assertEquals(original, new AppointmentBook(readBack));
+    //
```
Comment (Stratostorm) [inline src/test/java/seedu/address/storage/JsonAppointmentBookStorageTest.java:84]: Somemore leftover comments here

**Review by Stratostorm — APPROVED** [review]
Overall, okay to merge, if possible please remove the leftover comments

File: src/main/java/seedu/address/storage/Storage.java (hunk 1 above)
Comment (mechastriker3) [reply src/main/java/seedu/address/storage/Storage.java:86]: yea i think can

**Pull request merged**